.github/workflows/post-merge-matrix-deploy.yml

```diff
@@ -101,6 +105,10 @@ jobs:
             ARTIFACT_SOURCE_BUCKET_NAME_SECRET: KBV_BUILD_COMMON_CRI_ARTIFACT_SOURCE_BUCKET_NAME
             GH_ACTIONS_ROLE_ARN_SECRET:    KBV_BUILD_COMMON_CRI_GH_ACTIONS_ROLE_ARN
             SIGNING_PROFILE_NAME: KBV_BUILD_SIGNING_PROFILE_NAME
+          - target: CIC_BUILD
```

Comment:
I think target on line 94 also needs updated to include CIC_BUILD

Reply:
Fixed, thanks!